example/src/hello.go

```diff
@@ -1,6 +1,8 @@
 package main
 
-import "fmt"
+import (
```

Comment:
this file needs a lint violation so I can see that it's actually working. Can you suggest a short one?

Reply:
Added one.
```
INFO: From golangcilint src/golangcilint.hello_go.aspect_rules_lint.report:
../../bazel-source-roots/0/src/hello.go:13:2: SA1006: printf-style function with dynamic format string and no further arguments should use print-style function instead (staticcheck)
        fmt.Printf(hello)
```

Reply:
cool thanks